webgpu.h

```diff
@@ -964,6 +980,7 @@ typedef void (*WGPUProcCommandEncoderPopDebugGroup)(WGPUCommandEncoder commandEn
 typedef void (*WGPUProcCommandEncoderPushDebugGroup)(WGPUCommandEncoder commandEncoder, char const * groupLabel);
 typedef void (*WGPUProcCommandEncoderResolveQuerySet)(WGPUCommandEncoder commandEncoder, WGPUQuerySet querySet, uint32_t firstQuery, uint32_t queryCount, WGPUBuffer destination, uint64_t destinationOffset);
 typedef void (*WGPUProcCommandEncoderWriteTimestamp)(WGPUCommandEncoder commandEncoder, WGPUQuerySet querySet, uint32_t queryIndex);
+typedef void (*WGPUProcCommandEncoderFree)(WGPUCommandEncoder commandEncoder);
```

Comment:
Could we use the same naming suffix for the meaning of release/free for all objects?

Reply:
Basically using `Release` for all objects, just that sometimes objects don't have `Reference`?

Reply:
Yeah exactly

Reply:
Since these objects are special anyway, can't we just say encoder gets freed by wgpuCommandEncoderFinish, and pass encoders get freed by EndPass?

Reply:
Still should have a way to delete it (without errors) if you choose to never call Finish(). Finish() results in errors if you encoded something invalid.